### ISSUE TITLE
iOS Keeping old launch image after update

### ISSUE THREAD

**Issue by pashtetbezd** [jonbhanson/flutter_native_splash#215]
I have replaced launch image and run

`flutter pub run flutter_native_splash:create`

After restart the old app icon briefly flashes before switching back to my new one

`[✓] Flutter (Channel master, 2.6.0-6.0.pre.128, on macOS 11.5.2 20G95 darwin-x64, locale en-RU)
    • Flutter version 2.6.0-6.0.pre.128 at /Users/pussylogist/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision fa9790ca47 (2 hours ago), 2021-09-17 01:37:03 -0400
    • Engine revision e70febab63
    • Dart version 2.15.0 (build 2.15.0-118.0.dev)

[✓] Android toolchain - develop for Android devices (Android SDK version 31.0.0-rc4)
    • Android SDK at /Users/pussylogist/Library/Android/sdk
    • Platform android-S, build-tools 31.0.0-rc4
    • ANDROID_HOME = /Users/pussylogist/Library/Android/sdk
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 12.5.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.10.1

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2020.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.10+0-b96-7281165)

[✓] Connected device (2 available)
    • iPhone (mobile) • 2e04986891adcf3100ee982445c104859f2deb8b • ios            • iOS 14.7.1 18G82
    • Chrome (web)    • chrome                                   • web-javascript • Google Chrome 93.0.4577.82
    ! Error: iPhone is busy: Copying cache files from device. Xcode will continue when iPhone is finished. (code -10)

• No issues found!
Process finished with exit code 0`

**Comment by jonbhanson** [comment]
It is probably an [iOS splash caching bug](https://stackoverflow.com/questions/33002829/ios-keeping-old-launch-screen-and-app-icon-after-update), which can be solved by uninstalling your app, powering off your device, power back on, and then try reinstalling.

**Comment by pashtetbezd** [comment]
> It is probably an [iOS splash caching bug](https://stackoverflow.com/questions/33002829/ios-keeping-old-launch-screen-and-app-icon-after-update), which can be solved by uninstalling your app, powering off your device, power back on, and then try reinstalling.

restart device is not problem for me, but how can i explain it for end user in prod?

**Comment by jonbhanson** [comment]
The bug stems from iOS itself, not to this package or even Flutter.  So that would be a question for the iOS development team.

**Comment by felipe-xavier** [comment]
Actually it is happening to me on Android!

I changed the image for branding and now it just loads the background color.
I did:
- upgraded to 2.0.1+1
- flutter clean
- flutter pub get
- flutter pub run flutter_native_splash:remove
- flutter pub run flutter_native_splash:create
- Removed app
- Reboot device

And still nothing

I checked the image on `android/app/src/main/res/drawable-*/branding.png` and the image is updated, but after build, it does not appear on the splash screen.

What am I missing?
